### PR TITLE
Initial work to get native-image generation working on windows with graalvm 19

### PR DESCRIPTION
**Pull request by ctomc**
GraalVM 19.0.0 was released yesterday that comes with preview of native image generation for windows.

Entry point is native-image.cmd so maven plugin needs a small change to address that.

also "--no-server" flag is currently not present on windows version.
With this changes native compilation starts successfully but later on fails with some other issue. Probably related with early state of windows support.

### CODE REVIEW

File: core/creator/src/main/java/io/quarkus/creator/phase/nativeimage/NativeImagePhase.java [hunk 1]
```diff
@@ -345,12 +346,16 @@ public void provideOutcome(AppCreator ctx) throws AppCreatorException {
                     throw new AppCreatorException("GRAALVM_HOME was not set");
                 }
             }
-            nativeImage = Collections.singletonList(graalvmHome + File.separator + "bin" + File.separator + "native-image");
+            if (IS_WINDOWS) {
```
Comment (ebramirez) [inline core/creator/src/main/java/io/quarkus/creator/phase/nativeimage/NativeImagePhase.java]: Have you considered a condensed version?

Something like:
```
String nativeCommand = IS_WINDOWS ? "native-image.cmd" : "native-image";
nativeImage = Collections.singletonList(graalvmHome + File.separator + "bin" + File.separator + nativeCommand);
```

Comment (ctomc) [reply core/creator/src/main/java/io/quarkus/creator/phase/nativeimage/NativeImagePhase.java]: Sure, I just have a feeling that Windows related changes are just about add up going forward.

But I will update as suggested.

Comment (ctomc) [reply core/creator/src/main/java/io/quarkus/creator/phase/nativeimage/NativeImagePhase.java]: changed

**Review by dmlloyd — APPROVED** [review]

**Comment by dmlloyd** [comment]
Waiting for CI.

**Comment by dmlloyd** [comment]
Thanks @ctomc!

**Pull request merged**